ajax/libs/pipes-core/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "pipes-core",
+  "version": "0.8.4",
+  "description": "Basic utilities for web streams",
+  "filename": "dist/pipes.core.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pipesjs/core.git"
+  },
+  "keywords": [
+    "streams",
+    "pipesjs",
+    "utils"
+  ],
+  "author": "Diwank Singh <diwank.singh@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://pipes.js.org/core"
+}
```

Comment:
@extend1994 I was only using git as I thought it might be easier to start off with. But thank you for the clarification, switched to npm. Also, I was wondering if there's a convention for including es6 versions. Thanks again! Let me know if you think this needs further changes.